frontend/src/app/pages/advertisers-manage/advertisers-manage.component.html

```diff
@@ -72,3 +72,7 @@ <h4>Advertisements</h4>
 
   </div>
 </div>
+
```

Comment:
the benefit of the solution is that I reused 99% of the app-advertisers-add-advertisement component. I just had to add an Output to notifiy when the page has finished it process.

It could be improved though because we dont want to implement an output on EVERY page that would need to be in a modal.